website/docs/docs/workshops/bi-directional/pre_requisites.md

```diff
@@ -15,4 +15,9 @@ These instructions have been written for, and tested, in a *bash* (not zsh) shel
 
 :::info
 If you have access to a second screen, we recommend that you use it for this workshop, as there is a lot of window switching involved.
-:::
\ No newline at end of file
+:::
+
+:::info
+We `recommend following alone the instructions in the workshop,
```

Comment:
```suggestion
We recommend following alone the instructions in the workshop,
```

I suspect the backtick is erroneous?

Reply:
Is it meant to be "alone" or "along"?

Reply:
I  think I wanted to say "do it yourself" so you learn, rather than just cloning the repo